### PR TITLE
Fixes random floor generation logic

### PR DESCRIPTION
**Pull request by flosincapite**
It was possible for rooms to be omitted before.

Adds tests for floor.
Adds tests for some items.

### CODE REVIEW

**Comment by flosincapite** [comment]
Addresses #87

File: tests/unit/cronenbroguelike/test_floor.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    return no_visitados
+
+
+@patch("cronenbroguelike.rooms._rooms_for_theme")
+def test_all_rooms_reachable(parche_cámaras):
+    parche_cámaras.return_value = [_ImportantRoom] * 69 + [
+        rooms._CathedralCatacombs
+    ] * 420
+    numero_de_cámaras = 200
+    iglesia = floor.Floor.generate("cathedral", numero_de_cámaras)
```
Comment (xmunoz) [inline tests/unit/cronenbroguelike/test_floor.py]: iglesia != catedral

File: tests/unit/cronenbroguelike/test_floor.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    assert numero_de_cámaras == len(iglesia._rooms)
+    for _ in range(1):
```
Comment (xmunoz) [inline tests/unit/cronenbroguelike/test_floor.py]: 1?

File: tests/unit/cronenbroguelike/test_floor.py (hunk 1 above)
Comment (flosincapite) [reply tests/unit/cronenbroguelike/test_floor.py]: Sinónimos

**Review by xmunoz — APPROVED** [review]

**Pull request merged**